### ISSUE TITLE
loads_kw_is_net default value is True for builtin load profiles

### ISSUE THREAD

**Issue by NLaws** [NREL/REopt_Lite_API#68]
Issue:
The `loads_kw_is_net` input value defaults to `True` because it is assumed that most users that upload meter data for their load profile will have data that is net of PV production (gross load - PV). However, because `loads_kw_is_net=True` by default, when a user does not upload their load profile and uses a builtin profile, as well as specifies non-zero `existing_kw` for `PV`, the gross load gets calculated as the builtin profile plus the existing PV production, thus inflating the load profile beyond the DOE reference values. 

Proposed solution:
When using a builtin profile have the API change the `loads_kw_is_net` value to `False`.

**Comment by NLaws** [comment]
@t-kwasnik do you see any issue with changing `loads_kw_is_net` from `True` (the default) to `False` in the validator when:
1. `doe_reference_name` is provided, and
2. `PV.existing_kw` is non-zero?

**Comment by t-kwasnik** [comment]
@NLaws Nope, if there was a way to also make sure the user had supplied True and that wasn't filled in by a default that would be good so we don't override a defined input
